src/atomics.rs

```diff
@@ -0,0 +1,185 @@
+//! Conditionally implement traits for atomic types
+//!
+//! Note that we only implement traits for atomics types if the corresponding
+//! [`#[cfg(target_has_atomic)]`](https://doc.rust-lang.org/reference/conditional-compilation.html#target_has_atomic)
```

Comment:
Small nit: To be consistent with comment style used elsewhere:

```suggestion
//! [`#[cfg(target_has_atomic)]`][target_has_atomic]
```

...and then, at the bottom of the comment:

```rust
/// [target_has_atomic]: https://doc.rust-lang.org/reference/conditional-compilation.html#target_has_atomic
```

Reply:
Fixed!